layouts/partials/footer.html

```diff
@@ -47,5 +47,13 @@ <h5>About</h5>
     </footer>
     <script src="{{ "/_/js/site.js" | relURL }}"></script>
     <script src="{{ "/_/js/vendor/highlight.js" | relURL }}"></script>
+    <script type="application/ld+json">
+        {
+        "@context": "http://schema.org/",
+        "@type": "Organization",
+        "url": "https://camel.apache.org/staging/",
+        "logo": "https://camel.apache.org/staging/_/img/logo-d.svg"
```

Comment:
These URLs need to come from the [Hugo variables](https://gohugo.io/variables/site/), we won't be publishing to `/staging` for long, so best make that configurable. Probably best to use `.Site.BaseURL` here that's configured from the config.toml here:

https://github.com/apache/camel-website/blob/85632c2c1059b528d688d5e2bf1616b06722e9ba/config.toml#L1

Reply:
Hello Mr.Zoran, Can you check now?